packages/components/src/Payments/Payments.tsx

```diff
@@ -95,19 +117,26 @@ export class Payments extends React.Component<PaymentsProps, PaymentsStates> {
           paymentAdjustedEth={paymentAdjustedEth}
           selectedUsdToSpend={selectedUsdToSpend}
           etherToSpend={etherToSpend}
-          handleEditPaymentType={this.handleEditPaymentType}
+          renderContext={this.context.renderContext}
+          civilUser={currentUser}
+          handleBack={() => this.handleUpdateState(PAYMENT_STATE.SELECT_PAYMENT_TYPE)}
+          handleLogin={showWeb3Login}
+          handleLogout={logout}
         >
           <PaymentsEth
             postId={postId}
             newsroomName={newsroomName}
             paymentAddress={paymentAddress}
             shouldPublicize={shouldPublicize}
             userEmail={userEmail}
+            userChannelID={userChannelID}
             usdToSpend={usdToSpend}
-            handlePaymentSuccess={this.handleUpdateState}
             etherToSpend={this.state.etherToSpend}
             resetEthPayments={this.state.resetEthPayments}
+            context={this.context}
```

Comment:
Actually this in particular I'd just access context in the child component directly rather than passing it in.